File: backend/etterem-frontend/src/app/entity/Order.ts
export class Order {

  public fullname:String;
  public phonenumber: String;
  public comments: String;
  public tablenumber: number;
  public foods:String;


  constructor(fullname:String,phonenumber?:String, comments?: String,foods?:String,tablenumber?:number) {
    this.fullname=fullname;
    this.comments = comments || "";
    this.phonenumber=phonenumber || "00000000";
    this.tablenumber=tablenumber || 0;
    this.foods=foods;
  }
}

File: backend/src/main/java/szakdoga/backend/app/module/Order.java
package szakdoga.backend.app.module;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;

@Entity
@Table(name = "ORDERS")
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper = true)
@Data
public class Order extends BaseEntity {

    @Column
    private  String fullname;

    @Column
    private String phonenumber;

    @Column(nullable = false)
    private String comments;

    @Column
    private int tablenumber;

    @Column
    private String foods;

}
